integration_test/functions/src/auth-tests.ts

```diff
@@ -3,68 +3,82 @@ import * as functions from 'firebase-functions';
 import { expectEq, TestSuite } from './testing';
 import UserMetadata = admin.auth.UserRecord;
 
-export const createUserTests: any = functions.auth.user().onCreate((u, c) => {
-  const testId: string = u.displayName;
-  console.log(`testId is ${testId}`);
+const REGION = process.env.FIREBASE_FUNCTIONS_TEST_REGION || 'us-central1';
```

Comment:
This (and the `.region(REGION)` calls) are the only real changes in this file - the formatter just happened to move everything around slightly.